### ISSUE TITLE
Google TV Streamer won't work 

### ISSUE THREAD

**Issue by Detroit1980** [virresh/matvt#97]
I cannot activate accessibility on the Google TV Streamer. It appears Google may have blocked it. Matvt shows up in the accessibility options but as soon as I try turning it on, it turns itself back off. I even tried wireless adb and it also failed. If anybody has a solution, please let me know. Thanks

**Comment by curtiola** [comment]
Can confirm. The mouse toggle service just instantly turns off on this device.

**Comment by Racer159** [comment]
Sharing here since this worked for me on a different box - the onn 4k plus - but I believe it to be just related to Android 14:

First I checked
```bash
adb shell appops get io.github.virresh.matvt
```

And I saw `ACCESS_RESTRICTED_SETTINGS` was denied (this seems to be a new Android 14 feature)
```
Uid mode: POST_NOTIFICATION: ignore
SYSTEM_ALERT_WINDOW: allow
TOAST_WINDOW: allow; time=+40m16s299ms ago; duration=+2s520ms
BIND_ACCESSIBILITY_SERVICE: allow; time=+65d0h9m11s722ms ago
ACCESS_RESTRICTED_SETTINGS: deny; rejectTime=+7m48s565ms ago
```

But running:
```bash
adb shell appops set io.github.virresh.matvt ACCESS_RESTRICTED_SETTINGS allow
```

Did allow the toggle to work in the UI (for some reason trying to start the service with the commands in #18 didn't directly work though...)
